src/test/java/com/hocs/test/glue/MarkUpStepDefs.java

```diff
@@ -29,20 +31,32 @@
 
     @When("^I complete the markup stage$")
     public void completeTheMarkupStage() {
-        dataInput.selectTeam1();
-        successfulCaseCreation.clickSessionVariableViaLinkText();
+        homepage.findMyMarkupCase();
+        successfulCaseCreation.selectCaseReferenceNumberViaLinkText();
+        homepage.selectAllocationUserByVisibleText("Danny Large (danny.large@ten10.com)");
+        homepage.selectMyCases();
+        successfulCaseCreation.selectCaseReferenceNumberViaLinkText();
         markUpDecision.clickPolicyResponseRadioButton();
         page.clickContinueButton();
-        System.out.println("I have found the Add topic button, smashing left click repeatedly....");
         topics.clickAddTopicButton();
         topics.enterRealTopic();
+        try {
```

Comment:
i think you have this on the page class already
```
public void sleep(int milliseconds) {
        try {
            Thread.sleep(milliseconds);
        } catch (InterruptedException e) {
            e.printStackTrace();
        }
    }
```

so you can just write page.sleep(1000); here